src/components/common/topNav/TopNav.tsx

```diff
@@ -0,0 +1,167 @@
+import { styled } from "styled-components";
+
+import { Logo, IcSearch, IcNew } from "../../../assets/";
+
+function TopNav() {
+  return (
+    <TopNavWrapper>
+      <LeftContainer>
+        <LogoIcon />
+        <BtnContainer>
+          <NavBtn type="button">발견</NavBtn>
+          <NavBtn type="button">채용</NavBtn>
+          <NavBtn type="button">
+            성장/커뮤니티
+            <IcNewIcon />
+          </NavBtn>
+          <NavBtn type="button">에이전시</NavBtn>
+        </BtnContainer>
+      </LeftContainer>
+      <RightContainer>
+        <SearchBar>
+          <IcSearchIcon />
+          <SearchBarInput></SearchBarInput>
```

Comment:
Input 태그는 자식 태그를 가질 수 없으니까 셀프클로징 태그 사용해보는 건 어떨까요!!
```suggestion
          <SearchBarInput />
```